libs/coin-modules/coin-elrond/.unimportedrc.json

```diff
@@ -0,0 +1,4 @@
+{
+    "entry": [],
+    "ignoreUnimported": []
+}
```

Comment:
```suggestion
{
  "entry": ["src/index.ts", "src/bridge/js.ts", "src/network/index.ts", "src/signer.ts"],
  "ignoreUnimported": [
    "src/cli-transaction.ts",
    "src/datasets/elrond1.ts",
    "src/deviceTransactionConfig.ts",
    "src/formatters.ts",
    "src/helpers/denominate.ts",
    "src/helpers/handleTransactionStatus.ts",
    "src/helpers/hasMinimumDelegableBalance.ts",
    "src/helpers/randomizeProviders.ts",
    "src/specs.ts",
    "src/speculos-deviceActions.ts",
    "src/transaction.ts"
  ],
  "ignoreUnused": ["@ledgerhq/devices", "@ledgerhq/live-promise", "lodash", "expect", "invariant"]
}
```

Reply:
This will also fix some of the failing CI jobs.
https://github.com/LedgerHQ/ledger-live/actions/runs/10669818142/job/29572534016